custom-datetimes/datetimes.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+import pytz
+
+def date_parser(date):
+    if "/" in date:
```

Comment:
I like the generalizability that you're going for here by using a one-size-fits-all function. Usually looking at a single character to determine branching is a bit risky though.